cmd/nknc/commands/debug.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/nknorg/nkn/v2/api/httpjson/client"
+	"github.com/nknorg/nkn/v2/util/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// debugCmd represents the debug command
+var debugCmd = &cobra.Command{
+	Use:   "debug",
+	Short: "blockchain node debugging",
+	Long:  "With nknc debug, you could debug blockchain node.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		showusage := true
+		cmd.Flags().Visit(func(name *pflag.Flag) {
+			showusage = false
+		})
+		if showusage {
+			return cmd.Usage()
+		}
+		if err := debugAction(); err != nil {
+			log.Error(err)
```

Comment:
Why not return this err?

Reply:
because cobra would show the usage help text if I return err.

Reply:
> because cobra would show the usage help text if I return err.

Then this function will never return any err even it's return type is error?

Reply:
`cmd.Usage()` returns an error.